config/initializers/job_configurations.rb

```diff
@@ -195,6 +195,11 @@
         cron: cron_1st_of_mo,
         args: -> { [Time.zone.today] },
       },
+      monthly_key_metrics_report: {
+        class: 'Reports::MonthlyKeyMetricsReport',
+        cron: cron_1st_of_mo,
```

Comment:
looking at this and the one above:
- i think we should be trying to run every day, and then having stuff inside the job decide where to email vs not
- the comment on the one above says "weekly" but it's monthly

Reply:
I’ve addressed this for the bottom one and created https://cm-jira.usa.gov/browse/LG-11057 for the top one.